### DISCUSSION TITLE
Hide #_ADDBOOKINGFORM depending on date

### DISCUSSION THREAD

**Discussion by phutti** [liedekef/events-made-easy#216] [Q&A] [answered]
Hi Franky,

I have several events and want the booking start for all at a defined date.
I defined an eventfield -> fieldvalue "StartAnmeldung" where I define the Date, when booking will be possible

[eme_if tag='#_CURDATE' ge='#_FIELDVALUE{StartAnmeldung}'] #_ADDBOOKINGFORM [/eme_if]

This should make the bookingform visible after the defined booking start date, but this does not work.
The bookingform is always visible.

Any hints are welcome.

Kind regards
Peter

**Comment by liedekef** [comment, marked as answer]
#_CURDATE (without arguments) returns the date in localized format, meaning a string. Comparing strings with "greater than or equal to" will lead to unexpected results.

**Reply by phutti** [reply]
both dates formatted with Y-m-d works fine

thanks for the hint
